manifests/repo.pp

```diff
@@ -5,17 +5,21 @@
 # @example
```

Comment:
Probably good to add some description now that there's more than 1 example. Something like:
```suggestion
# @example Inclusion using defaults
```